formulas/wick-base/functions/wick-make-group

```diff
@@ -0,0 +1,37 @@
+#!/usr/bin/env bash
+# Create or manage a group on the system.
+#
+# $1            - Groupname to create on the system
+#
+# If the given group already exists `groupadd` will not  be run.
+# This can be used in conjunction with wickMakeUser to assign
+# users to common groups.
+#
+# Examples
+#
+#   # Create a normal group.
```

Comment:
What does a non-normal group look like?

Reply:
You amended this to say "a unix group" but Unix is a proper noun and deserves a capital U. Or you can omit the descriptor entirely and say "Create a group".